nntrainer/layers/conv2d_layer.cpp

```diff
@@ -66,16 +68,16 @@ static void col2im(const Tensor &col_matrix, const TensorDim &kdim,
   unsigned wdilation = dilation[1];
 
   /// image considering padding
-  unsigned im_eff_height = im_height + 2 * ph;
-  unsigned im_eff_width = im_width + 2 * pw;
+  unsigned im_eff_height = im_height + pt + pb;
```

Comment:
i recommend to make inline function for these.

Reply:
I have a little clean up idea for this things, let me delay this until v2 is merged :)

Reply:
cool.